app/src/main/java/hm/orz/chaos114/android/tumekyouen/modules/kyouen/KyouenActivity.kt

```diff
@@ -194,29 +196,26 @@ class KyouenActivity : DaggerAppCompatActivity(), KyouenActivityHandlers {
         binding.fragmentContainer.addView(tumeKyouenView)
 
         oldView.animate()
-                .translationX(oldTranslationX)
-                .setDuration(250).interpolator = AccelerateInterpolator()
+            .translationX(oldTranslationX)
+            .setDuration(250).interpolator = AccelerateInterpolator()
         tumeKyouenView.animate()
-                .translationX(0f)
-                .setDuration(250)
-                .setInterpolator(AccelerateInterpolator())
-                .setListener(object : Animator.AnimatorListener {
-                    override fun onAnimationStart(animation: Animator) {
-
-                    }
-
-                    override fun onAnimationEnd(animation: Animator) {
-                        binding.fragmentContainer.removeView(oldView)
-                    }
+            .translationX(0f)
+            .setDuration(250)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/kyouen-android/blob/f44fa79828c9d915587d9b3fabd8550e24815e45/app/src/main/java/hm/orz/chaos114/android/tumekyouen/modules/kyouen/KyouenActivity.kt#L203"></span>This expression contains a magic number. Consider defining it to a well named constant.</td>
    </tr>
  </tbody>
</table>